_config.yml

```diff
@@ -231,3 +231,9 @@ version_messages:
     minor: "마지막 갱신 이후로 원본 문서에 작은 변경사항이 있습니다."
     patch: "마지막 갱신 이후로 원본 문서에 사소한 변경사항이 있습니다."
     none: "이 번역은 최신입니다."
+  pl:
+    outdated: "Niektóre treści tego tłumaczenia mogą być nieaktualne."
+    major: "Kilka ważnych zmian wprowadzono do pierwotnej lekcji od ostatniej aktualizacji."
+    minor: "Kilka drobnych zmian wprowadzono do pierwotnej lekcji od ostatniej aktualizacji."
+    patch: "Kilka poprawek wprowadzono do pierwotnej lekcji od ostatniej aktualizacji."
```

Comment:
Thanks a lot for taking your time to consider this, @Koziolek.
I've just updated the translation according to your comment. I am learning Polish little by little , probably it's still too early for me to participate in this kind of translation work.